src/sidebar/components/thread-list.js

```diff
@@ -72,37 +70,32 @@ function ThreadList({ thread, $rootScope }) {
     [topLevelThreads, threadHeights, scrollPosition, scrollContainerHeight]
   );
 
-  // Listen for when a new annotation is created in the application, and trigger
-  // a scroll to that annotation's thread card (unless highlight or reply)
+  // Scroll to newly created annotations and replies.
+  const newAnnotationTag = useStore(store => {
+    const ann = store
+      .unsavedAnnotations()
+      .find(ann => !ann.id && !isHighlight(ann));
+    return ann ? ann.$tag : null;
```

Comment:
Wait: I'm confused—what if there are multiple new annotations?

Reply:
We skip any that are highlights and only scroll to the first non-highlight annotation. One issue your question did raise though is that I'm not sorting the annotations in any way, so the definition of "first" is ill-defined. I'll fix that.

Reply:
I took another look at this and made the behavior when there are multiple unsaved annotations clearer. See the commit message for details.